launch/rcnn_moped.launch

```diff
@@ -0,0 +1,8 @@
+<launch>        
```

Comment:
Add a short comment or url to rcnn_moped here. Add an instruction to README.

Reply:
https://github.com/personalrobotics/rcnn_moped/tree/master